net/goeap_proxy/Makefile

```diff
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=goeap_proxy
+PKG_VERSION:=0.3.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:= Gabriel Rodriguez <lifehacksback@gmail.com>
+PKG_LICENSE:=none
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/pyther/goeap_proxy.git
+PKG_MIRROR_HASH:=e4014a8dc317d3325d02c1ae81678d6552628bd7a3e253c8ad01a6912ec49dbf
+PKG_SOURCE_VERSION:=3abd66cf35c70b86fe75989ca64c7a72cc8fa797
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/pyther/goeap_proxy
+GO_PKG_LDFLAGS_X:=main.Version=$(PKG_VERSION) main.BuildStamp=$(shell date +%FT%T%z)
```

Comment:
This is not reproducible

Reply:
What do you mean?

Reply:
Because the value for `main.BuildStamp` depends on the build date/time, if you compile this at two different times with the exact same source code you will get two slightly different binaries, hence not reproducible.